src/components/icons/index.js

```diff
@@ -330,7 +330,7 @@ export const SpeedometerIcon = createStyledIcon(
 export const SplitIcon = createStyledIcon(OrigSplitIcon, 'SplitIcon');
 export const StackIcon = createStyledIcon(OrigStackIcon, 'StackIcon');
 export const StarIcon = createStyledIcon(OrigStarIcon, 'StarIcon');
-export const SuplinkIcon = createStyledIcon(OrigSuplinkIcon, 'SuplinkIcon');
+export const SubLinkIcon = createStyledIcon(OrigSubLinkIcon, 'SubLinkIcon');
```

Comment:
I am not sure I 100% like the name. We try to name by "what is is" not "what it's for". So maybe this should be SubArrow? Sorry. @montezume should say.

Reply:
it was `sublink` (actually `suplink`) from the SVG

Reply:
![image](https://user-images.githubusercontent.com/7680511/57232245-77a1d300-701c-11e9-9050-a1334df2f58b.png)

Reply:
![image](https://user-images.githubusercontent.com/7680511/57232275-88eadf80-701c-11e9-8dbf-2e47d42d3ad9.png)